### ISSUE TITLE
docker container: Is ZWAVEJS_DISABLE_SOFT_RESET still a valid env var?

### ISSUE THREAD

**Issue by josh-m-sharpe** [zwave-js/zwave-js-ui#2776]
If I try to "soft reset", Iget this error:
![Z-Wave_JS_UI](https://user-images.githubusercontent.com/39473/199028731-f7743244-aae1-4da1-acab-bbc446b0a667.jpg)

I've set:
```
environment:
- ZWAVE_DISABLE_SOFT_RESET=false
```

I've also tried `'false'` and `true` just to see if they have any effect.  Stopped/started the container between changes.

Nothing appears to have any effect.  Thanks!

**Comment by josh-m-sharpe** [comment]
Hmm, just noticed this in my logs:
```
zwavejs2mqtt    | 14:47:07.963 DRIVER   Soft reset is enabled through config, but this stick does not support it.
```
... pops out amongst much other when I save anything in settings.  *Something* is restarting, not sure what though.

**Comment by jmgiaever** [comment]
There is a case where zwjs thinks a controller doesn't support soft reset. 

Can't remember when it might start to think so, but I might be in cases where a stick gets stuck and zwjs tries to start up indefinitely, and you have to shutdown and unplug the stick for it to act up again. 

Then it will be saved in one of the files for zwjs that it's unsupported and it will prevent you from using soft reset, either way. 

The fix is to manually edit this file and flip `false` to `true`. I can't remember which file exactly, but it should be within the `json` or `jsonl` files with the home hex id. Search for the word "soft" or "reset" and see what you'll find. 

**However**.... This requires your stick to actually support soft reset.

**Comment by robertsLando** [comment]
> There is a case where zwjs thinks a controller doesn't support soft reset.

Is @AlCalzone aware of this?

**Comment by AlCalzone** [comment]
Yes, the driver remembers when a stick doesn't "come back" After a soft reset. E.g. When the stick might support it, but the environment isn't configured for it (VM).

The file is the <homeid>.jsonl, what @jmgiaever wrote applies.
